asm/macros/event.inc

```diff
@@ -765,7 +765,7 @@
 	.2byte \y
 	.endm
 
-	.macro moveobjectoffscreen index:req
+	.macro copyobjectxytoperm index:req
```

Comment:
Could you explain the reasoning behind this name change?

Reply:
Explained in the commit message:
Renaming the command to something that makes more sense as to what the command actually does. The command copies the current xy of the instantiated object back to its template, so that the object remains in the same place if the player walks away and despawns the object, and then comes back. "moveobjectoffscreen" implies that it might be taking the object and moving it physically to an off-screen location somehow.